Package: acpica-unix
Severity: important
Version: 20060912-2
Tags: patch


Hi,

the current version fails to build on GNU/kFreeBSD.

It needs small tweak to define uintptr_t, see below.

It would also be nice if you can ask upstream
to include this change.

Thanks in advance

                         Petr


---
 source/include/platform/acenv.h |    3 +++
 1 file changed, 3 insertions(+)

Index: acpica-unix-20191213/source/include/platform/acenv.h
===================================================================
--- acpica-unix-20191213.orig/source/include/platform/acenv.h
+++ acpica-unix-20191213/source/include/platform/acenv.h
@@ -178,6 +178,9 @@
 #endif
 
 /*! [Begin] no source code translation */
+#if defined(__GLIBC__)
+#include <stdint.h>
+#endif
 
 /******************************************************************************
  *
